system/lib/compiler-rt/lib/sanitizer_common/sanitizer_linux.cpp

```diff
@@ -483,7 +483,9 @@ uptr internal_execve(const char *filename, char *const argv[],
 
 #if !SANITIZER_NETBSD
 void internal__exit(int exitcode) {
-#if SANITIZER_FREEBSD || SANITIZER_SOLARIS
+#if SANITIZER_EMSCRIPTEN
+  __wasi_proc_exit(exitcode);
+#elif SANITIZER_FREEBSD || SANITIZER_SOLARIS
   internal_syscall(SYSCALL(exit), exitcode);
```

Comment:
I guess it's unlikely that we have users that call syscalls like this, but what do you think about a changelog mention of this?

Reply:
We have not generally made changelog entries for syscall naming changes as far as I am aware.

Reply:
I guess back in v1.39.12 I wrote: `Rename syscalls to have meaningful names (#10750).` .. but that was after renaming *all* of them at once :)